src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/ampSrcBuilder.js

```diff
@@ -1,7 +1,10 @@
 import Url from 'url-parse';
 
 const ampSrcBuilder = href => {
-  const ampSrcPrefix = process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN;
+  const ampSrcPrefix =
+    process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN === 'http://localhost:7080'
+      ? 'https://news.test.files.bbci.co.uk'
+      : process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN;
```

Comment:
So we need this because `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` points to localhost [in local.env](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3). That makes sense because Simorgh hosts our static assets on a local build.

In this case we a referencing static assets that Simorgh does not manage, I'd personally go back to adding a specific `.env` var for includes as on this PR: https://github.com/bbc/simorgh/pull/6873#discussion_r442248006 ; there is a distinction between static assets Simorgh manages and those managed elsewhere. I'd chat with @hindsc52 first if you think we should go down this route as it was his suggestion to use this var.

Reply:
ahhh I did not realise on the last PR that we would want to use the test environment on localhost. In that case a separate .env variable would be better and would remove the required logic here.